sunkit_instruments/suvi/io.py

```diff
@@ -0,0 +1,320 @@
+
+    IS_COMPOSITE = False
+
+    # If it is a fits file, it is easy!
+    if input_filename.lower().endswith(fits_file_extensions):
+        # Based on the filename, determine if we are dealing
+        # with L1b files or HDR composites (or neither).
+        if any(fn in os.path.basename(input_filename) for fn in composite_matches):
+            if return_header_only:
+                header = fits.getheader(input_filename, 1)
+            else:
+                hdu = fits.open(input_filename)
+                data, header = hdu[1].data, hdu[1].header
+                hdu.close()
+            IS_COMPOSITE = True
+        elif any(fn in os.path.basename(input_filename) for fn in L1b_matches):
+            hdu = fits.open(input_filename)
+            if return_header_only:
+                header = fix_L1b_header(input_filename)
+            else:
+                hdu = fits.open(input_filename)
+                data, header = hdu[0].data, fix_L1b_header(input_filename)
+            if return_DQF:
+                dqf = hdu[1].data
+            hdu.close()
+        else:
+            raise ValueError("File "+input_filename+" does not look like a SUVI L1b FITS file or L2 HDR composite.")
+
+    # If it is a netCDF file on the other hand...
+    elif input_filename.lower().endswith(netCDF_file_extensions):
+        # This is how the global attributes of the netCDF file get
+        # mapped to the corresponding FITS header keywords.
+        tag_mapping = {'instrument_id':           'INST_ID',
+                       'platform_ID':             'TELESCOP',
+                       'instrument_type':         'INSTRUME',
+                       'project':                 'PROJECT',
+                       'institution':             'ORIGIN',
+                       'production_site':         'PRODSITE',
+                       'naming_authority':        'NAMEAUTH',
+                       'production_environment':  'PROD_ENV',
+                       'production_data_source':  'DATA_SRC',
+                       'processing_level':        'LEVEL',
+                       'algorithm_version':       'CREATOR',
+                       'title':                   'TITLE',
+                       'keywords_vocabulary':     'KEYVOCAB',
+                       'date_created':            'DATE',
+                       'orbital_slot':            'ORB_SLOT',
+                       'dataset_name':            'FILENAME',
+                       'iso_series_metadata_id':  'ISO_META',
+                       'id':                      'UUID',
+                       'LUT_Filenames':           'LUT_NAME',
+                       'license':                 'LICENSE',
+                       'keywords':                'KEYWORDS',
+                       'summary':                 'SUMMARY'}
+
+        # Mapping for the FITS header keywords and their comments
+        tag_comment_mapping = {'SIMPLE':    'file does conform to FITS standard'             , \
```

Comment:
Same for this, to be moved to the top and renamed to be upper case.

You also do not need the `\` at the end of each line.

Reply:
I know I don't need it, I just like it as a visual indicator. Does that violate the codestyle rules?

Reply:
I am not aware of it violating any code style rule, it just isn't required to break new lines when you have a valid `,`